packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -1,412 +1,74 @@
-<%
-if (answers.isUsingTypeScript && answers.purpose === 'electron') {
-    %>/// <reference types="wdio-electron-service" />
-<% }
-
-if (answers.isUsingTypeScript && answers.serenityAdapter) {
-    %>import type { WebdriverIOConfig } from '@serenity-js/webdriverio'
-<% }
-
-if (answers.isUsingTypeScript) {
-    %>export const config: <%= answers.serenityAdapter ? 'WebdriverIOConfig' : 'WebdriverIO.Config' %> = {<%
-} else if (answers.esmSupport) {
-    %>export const config = {<%
-} else {
-    %>exports.config = {<%
-} %>
-    //
-    // ====================
-    // Runner Configuration
-    // ====================
-    // WebdriverIO supports running e2e tests as well as unit and component tests.
-    runner: <%
-    if (answers.runner === 'browser') {
-        if (answers.preset) {
-            %>['browser', {
-        preset: '<%- answers.preset %>',
-        // start browser window when `DEBUG` environment variable is set
-        headless: !Boolean(process.env.DEBUG)
-    }]<%
+<% if (answers.isUsingTypeScript && answers.purpose === 'electron') { %>
+    /// <reference types="wdio-electron-service" />
+    <% }-%>
```

Comment:
The reason why it is formated this way:
```
if (answers.isUsingTypeScript && answers.purpose === 'electron') {
    %>/// <reference types="wdio-electron-service" />
<% }
```
is so that `/// <reference types="wdio-electron-service" />` starts at the beginning of the line, not after 4 spaces

Reply:
Good catch, let me correct this